code/datums/gamemodes/pod_wars.dm

```diff
@@ -0,0 +1,3106 @@
+#define TEAM_NANOTRASEN 1
```

Comment:
PLEASE consider splitting this up into a folder

Reply:
Yeah, I should shouldn't I? I don't know why I haven't, just being lazy I guess. It was starting to get _really_ messy/annoying at the end.